components/create_comment/create_comment.jsx

```diff
@@ -211,6 +218,13 @@ class CreateComment extends React.PureComponent {
          * Should preview be showed
          */
         shouldShowPreview: PropTypes.bool.isRequired,
+
+        /*
+            Group member mention
+        */
+        selectChannelMemberCountsByGroup: PropTypes.func.isRequired,
+        allowReferencedGroups: PropTypes.array,
```

Comment:
the wording of this is kinda weird it makes me think it should be a boolean maybe something like `groupsWithAllowReference` would make more sense as a name? 0/5 on renaming it though

Reply:
Good idea! Done.